concertfit/Controllers/HomeController.cs

```diff
@@ -6,12 +6,15 @@
 using System.Threading.Tasks;
 using concertfit.Models;
 using Microsoft.AspNetCore.Mvc;
+using SpotifyArtistsResponse;
 
 // For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
 
 namespace concertfit.Controllers
 {
-    [Route("api")]
+    
+    [Route("TicketMaster")]
```

Comment:
This route looks more descriptive... and the BindProperties allows you to do more in Get.  Good suggestion.